gnd/src/main/java/com/google/android/gnd/ui/offlinebasemap/OfflineBaseMapsFragment.java

```diff
@@ -66,6 +66,9 @@ public View onCreateView(
     recyclerView.setAdapter(offlineBaseMapListAdapter);
 
     viewModel.getOfflineAreas().observe(getViewLifecycleOwner(), offlineBaseMapListAdapter::update);
+    viewModel
```

Comment:
This can now be set by binding the noAreasDownloadedMessage in the fragment layout. Example:
https://github.com/google/ground-android/blob/f1d82b94704990b49c21838d78a4163518fd07dc/gnd/src/main/res/layout/edit_observation_frag.xml#L92

Reply:
c6b1a83